web/src/main/ui/app/components/flows-new/edit-flow/edit-flow.component.ts

```diff
@@ -180,7 +174,6 @@ export class EditFlowComponent implements OnInit, OnDestroy {
   }
   updateStep(step) {
     this.manageFlowsService.updateStep(this.flow.id, step.id, step).subscribe(resp => {
-      this.editFlowUi.stepUpdated(step);
```

Comment:
The stepUpdated() was added as part of the following: https://github.com/marklogic/marklogic-data-hub/pull/2689
I don't think it should be removed.